src/amuse/units/core.py

```diff
@@ -83,12 +83,9 @@ def __truediv__(self, other):
 #            return factor_unit(1.0 / other, self)
 
     def __rmul__(self, other):
-        if other == 1:
-            return self
-        else:
-            if isinstance(other, unit):
-                return factor_unit(other, self)
-            return self.new_quantity(other)
+        if isinstance(other, unit):
+            return factor_unit(other, self)
```

Comment:
this is wrong, because factor units are supposed to construct from a number(like) and a unit..you should only get here with other a number..

Reply:
I see

Reply:
Replaced it with mul_unit, I think that's correct?